packages/lit-analyzer/package.json

```diff
@@ -26,14 +26,55 @@
 	"scripts": {
 		"clean": "rimraf lib",
 		"prepare": "npm run build",
-		"build": "npm run clean && npm run build:ts && npm run check-version",
-		"build:ts": "tsc",
-		"watch": "tsc --watch",
+		"build": "wireit",
 		"readme": "readme generate -i readme.blueprint.md -c readme.config.json",
-		"test": "ava --color",
-		"test:all": "ava --color --fail-fast",
+		"test": "wireit",
+		"test:ava": "wireit",
+		"test:all": "npm test",
 		"test:watch": "ava --watch --color",
-		"check-version": "node scripts/check-version.js"
+		"check-version": "wireit"
+	},
+	"wireit": {
+		"build": {
+			"command": "tsc --build",
+			"files": [
+				"src/**/*",
+				"tsconfig.json"
+			],
+			"output": [
+				"lib",
+				"test",
+				"scripts",
+				"index.js",
+				"index.d.ts",
+				"index.d.ts.map",
+				"tsconfig.tsbuildinfo"
+			],
+			"clean": false
```

Comment:
Recommend `"clean": "if-file-deleted"` (needs `wireit@0.1.1`).

See https://github.com/google/wireit#typescript for explanation

Reply:
Done, that's nicer